src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -344,6 +348,25 @@ private static void checkGenomicsDBAgainstExpected(final String workspace, final
         return results.iterator();
     }
 
+    @Test
+    public void testSampleNameWithSpaces() throws IOException {
+        final File outOfOrderSampleMap = getSampleMapFile(
+                "HG00268 withSpaces\t" + HG_00268_withSpaces + "\n" +
+                        "NA19625\t" + NA_19625 + "\n" +
+                        "HG00096\t" + HG_00096 );
+
+        final String workspace = createTempDir("gendbtest").getAbsolutePath() + "/workspace";
+
+        ArgumentsBuilder args = new ArgumentsBuilder()
+                .addArgument(GenomicsDBImport.BATCHSIZE_ARG_LONG_NAME, String.valueOf(2))
+                .addFileArgument(GenomicsDBImport.SAMPLE_NAME_MAP_LONG_NAME, outOfOrderSampleMap).addArgument("L", IntervalUtils.locatableToString(TRUNCATED_INTERVAL))
+                .addArgument(GenomicsDBImport.WORKSPACE_ARG_LONG_NAME, workspace);
+
+        runCommandLine(args);
```

Comment:
Is there a reason you can't just use the pre-existing `writeToGenomicsDB()` method here?

Reply:
Because that doesn't take a sample name map. Thus to test the support for whitespaces in the sample name map I must use this codepath.